Tools/autotest/autotest.py

```diff
@@ -426,18 +426,17 @@ def run_tests(steps):
                 passed = False
                 failed.append(step)
                 results.add(step, '<span class="failed-text">FAILED</span>', time.time() - t1)
-                continue
+            else:
```

Comment:
Please invert the check on this `if ...` call; so not `if not run_step(step)` but `if run_step(step)`.  Into the future I'd like to make `run_step` void.